frontend/public/components/storage/create-pvc.tsx

```diff
@@ -36,6 +36,9 @@ const provisionerAccessModeMapping = {
   'kubernetes.io/storageos' : ['ReadWriteOnce'],
 };
 
+// These provisioners not support ROX OCS 4.2
+const cephStorageProvisioners = ['ceph.rook.io/block', 'cephfs.csi.ceph.com', 'rbd.csi.ceph.com'];
+
```

Comment:
Added these separately because the complete provisioner string might start with NAMESPACE, which is a variable. Hence, unlike provisioners given above, we have to compare strings in a different way.